crates/css_ast/src/types/palette_identifier.rs

```diff
@@ -0,0 +1,4 @@
+use css_parse::T;
+
+// https://www.w3.org/TR/css-fonts-4/#typedef-font-palette-palette-identifier
+pub type PaletteIdentifier = T![DashedIdent];
```

Comment:
It would be helpful to use the [new type idiom](https://doc.rust-lang.org/rust-by-example/generics/new_types.html) here, because then we can `impl` on top of this (for example, later on we can track whether or not a PaletteIdentifier is used in an `@font-face` rule). I believe the current `pub type` will fall away meaning we cannot distinguish between `T![DashedIdent]` and a `PaletteIdentifier`.


```suggestion
pub struct PaletteIdentifier(T![DashedIdent]);
```

This will mean implementing `parse` etc but perhaps we can add a `derive(Parse)` for such simple wrappers?

Reply:
I might undo this from this PR, and tackle that in a separate diff